pkg/initapi/validate.go

```diff
@@ -48,3 +49,10 @@ func validateSpotPriceOnlyForAwsCluster(initCluster *cluster.Cluster) error {
 	}
 	return nil
 }
+
+func validateClusterNameOnlyForDOCluster(initCluster *cluster.Cluster) error {
```

Comment:
So the function names here are documentation, as we print the name of the function if something fails so the function name should let the user know that is wrong and what they would need to fix in order to succeed.

So maybe `validateDigitalOceanClusterNameDoesNotContainPeriod` or something would be better? so the user would know what they need to fix

Reply:
@kris-nova I updated the function name to `validateDOClusterNameDoesNotContainPeriod`. Does it sound better?